app/src/main/resources/application-local.yml

```diff
@@ -3,12 +3,6 @@
 #
 # File application-nonprod.yml will also be loaded due to the spring.profiles.group configuration in application.yml
 
-vro:
-  persist:
-    tracking:
-      enabled: true
-      base_folder: /tmp/persist/tracking
-
```

Comment:
When `app/src/docker/docker-compose.yml` sets ENV=local, this file is used. We set a reasonable default in `app/src/main/resources/application.yml` (https://github.com/department-of-veterans-affairs/abd-vro/pull/1243/files#diff-cfaac170cb673164b4d30a6ae0f20631762be4126821b99e86a4421f09bf4fdaR185), so this is not needed.